students/fortucj/lesson03/mailroom.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+
+"""
+The Mailroom assignment.
+
+The data is a global list.
+
+Appending donations was repetitive, so was made into its own function 'add_donations()'
+
+As the assignment becomes more complex, parts may be broken into separate functions.
+"""
+
+import sys
+
+# This was originally a list of lists.  I changed it based on the recommendation in the notes.
+data = [('Bob', [17.56]), ('Billy', [500.00, 1000.00]), ('Joe Schmoe', [2.00, 0.03, 45.00]), ('This Guy', [1.00, 100000]), ('That Gal', [9876.54])]
+
+# Generate a list of names to make looping easier.
+Name_List = []
```

Comment:
Variables should be snake_case in Python - basically everything except class names.  name_list